code/modules/tgui/params.dm

```diff
@@ -0,0 +1,133 @@
+/datum/params
+	VAR_PRIVATE/_unsafe_params
+
+/datum/params/New(unsafe_params)
+	. = ..()
+	_unsafe_params = unsafe_params
+
+/datum/params/Destroy(force, ...)
+	_unsafe_params = null
+	. = ..()
+
+/datum/params/can_vv_get(var_name)
+	// You can't look at this variable as it contains potentially unsafe to render things
+	// which may be poorly handled by VV or rendered in chat if someone attempts to edit it.
+	// It's safer if we just leave this alone.
+	if (var_name == "_unsafe_params")
+		return FALSE
+	return ..()
+
+// =============================
+// Boolean Handling
+// =============================
+
+/// Returns true if the requested parameter is equal to the value
+/// Returns false otherwise
+/datum/params/proc/is_param_equal_to(param, value)
+	return _unsafe_params[param] == value
+
+/// Returns the requested parameter as either a true or false value depending
+/// on the truthyness of the parameter.
+/datum/params/proc/get_boolean(param)
+	return !!_unsafe_params[param]
+
+// =============================
+// Text Handling
+// =============================
+
+/// Returns the requested parameter as fully sanitised text, removing \n and \t as well as encoding HTML.
+/datum/params/proc/get_sanitised_text(param)
+	return sanitize_text(_unsafe_params[param])
```

Comment:
```suggestion
	return sanitize(_unsafe_params[param])
```

sanitize_text only checks if the input is text

Reply:
Ah yea, you are correct. Good spot